pyproject.toml

```
@@ -15,7 +15,7 @@ classifiers = [
     "Programming Language :: Python",
 ]
 dependencies = [
-    "napari[all]>=0.4.19",
+    "napari[all]>=0.5.4",
```

Not necessary with the suggestion above.

```suggestion
    "napari[all]>=0.4.19",
```